modules/view-network-policies.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-network-policies.adoc
+:_module-type: CONCEPT
+[id="view-network-policies_{context}"]
+= Viewing network policies
+
+[role="_abstract"]
+Network policies specify how groups of pods are allowed to communicate with each other and with other network endpoints.
+Kubernetes `NetworkPolicy` resources use labels to select pods and define rules that specify what traffic is allowed to or from the selected pods.
+{product-title} discovers and displays network policy information for all your Kubernetes clusters, namespaces, deployments, and pods, in the *Network Graph* view.
+
+To view network policies and other related details for deployments in a namespace, you can select the namespace in *Network Graph* view.
+
+The namespace details panel lists all deployments in the selected namespace.
+You can then hover over a deployment in the details panel and select the *Navigate to deployment* (arrow) icon that appears on the right to view deployment details.
+
+Alternatively, you can directly select a deployment in the *Network Graph* view to view its details.
+The deployment details panel includes the *Network Flows*, *Details*, and *Network Policies* tabs.
+
+You can select each tab to view related information.
+* The *Network Flows* tab shows information about ingress and egress connections, protocols, and port numbers in use for that deployment.
```

Comment:
This still needs a blank line before these bullets in order for them to render properly